packages/opentelemetry-plugin-xml-http-request/test/xhr.test.ts

```diff
@@ -596,4 +596,71 @@ describe('xhr', () => {
       assert.strictEqual(events.length, 12, 'number of events is wrong');
     });
   });
+
+  describe('should work when async=false', () => {
+    let webTracerWithZone: api.Tracer;
+    let webTracerProviderWithZone: WebTracerProvider;
+    let dummySpanExporter: DummySpanExporter;
+    let exportSpy: any;
+    let rootSpan: api.Span;
+    const url =
+      'https://raw.githubusercontent.com/open-telemetry/opentelemetry-js/master/package.json';
+
+    beforeEach(done => {
+      const resources: PerformanceResourceTiming[] = [];
+      resources.push(
+        createResource({
+          name: url,
+        })
+      );
+      webTracerProviderWithZone = new WebTracerProvider({
+        logLevel: LogLevel.ERROR,
+        plugins: [new XMLHttpRequestPlugin({})],
+      });
+      webTracerWithZone = webTracerProviderWithZone.getTracer('xhr-test');
+      dummySpanExporter = new DummySpanExporter();
+      exportSpy = sinon.stub(dummySpanExporter, 'export');
+      webTracerProviderWithZone.addSpanProcessor(
+        new tracing.SimpleSpanProcessor(dummySpanExporter)
+      );
+
+      rootSpan = webTracerWithZone.startSpan('root');
+      webTracerWithZone.withSpan(rootSpan, () => {
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', url, false);
```

Comment:
Don't think this should be blocking, but I think this test could be combined with the existing ones by modifying `getData(...)` to accept a new `async` parameter? This way everything that is tested with async is also tested without it

Reply:
In theory I think so, but I was unsure if the sinon fake xhr behaved properly with async=false so I figured I'd use the raw API.  Otherwise, yes, it is a bit of a copy+paste job from a previous test.  I'll spend a bit of time looking into this idea.

Reply:
It looks based on a quick hack that using async:false can work with all existing tests.  I'm unfamiliar with Karma: how can I cleanly get the equivalent of, say, Spock's "where" table here?

Reply:
Nevermind, found an example in mocha docs.

Reply:
This worked out nicely; now running all the tests with both async=true and async=false.